### ISSUE TITLE
Fix duplicated dependency

### ISSUE THREAD

**Issue by l00k** [Phala-Network/js-sdk#25]
```
yarn why v1.22.19
[1/4] Why do we have the module "@polkadot/wasm-crypto"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "@polkadot/wasm-crypto@6.3.1"
info Reasons this module exists
   - "@phala#sdk" depends on it
   - Hoisted from "@phala#sdk#@polkadot#wasm-crypto"
=> Found "@polkadot/util-crypto#@polkadot/wasm-crypto@6.4.1"
info This module exists because "@phala#sdk#@polkadot#util-crypto" depends on it.
```

This leads to errors while using api
```
@polkadot/wasm-crypto has multiple versions, ensure that there is only one installed.
Either remove and explicitly install matching versions or dedupe using your package manager.
The following conflicting packages were found:
        cjs 6.3.1       node_modules/@polkadot/wasm-crypto/cjs
        cjs 6.4.1       node_modules/@polkadot/util-crypto/node_modules/@polkadot/wasm-crypto/cjs
@polkadot/wasm-bridge has multiple versions, ensure that there is only one installed.
Either remove and explicitly install matching versions or dedupe using your package manager.
The following conflicting packages were found:
        cjs 6.3.1       node_modules/@polkadot/wasm-bridge/cjs
        cjs 6.4.1       node_modules/@polkadot/util-crypto/node_modules/@polkadot/wasm-bridge/cjs
@polkadot/wasm-crypto-wasm has multiple versions, ensure that there is only one installed.
Either remove and explicitly install matching versions or dedupe using your package manager.
The following conflicting packages were found:
        cjs 6.3.1       node_modules/@polkadot/wasm-crypto-wasm/cjs
        cjs 6.4.1       node_modules/@polkadot/util-crypto/node_modules/@polkadot/wasm-crypto-wasm/cjs
@polkadot/wasm-util has multiple versions, ensure that there is only one installed.
Either remove and explicitly install matching versions or dedupe using your package manager.
The following conflicting packages were found:
        cjs 6.3.1       node_modules/@polkadot/wasm-util/cjs
        cjs 6.4.1       node_modules/@polkadot/util-crypto/node_modules/@polkadot/wasm-util/cjs
```

**Comment by vabaly** [comment]
> ```
> yarn why v1.22.19
> [1/4] Why do we have the module "@polkadot/wasm-crypto"...?
> [2/4] Initialising dependency graph...
> [3/4] Finding dependency...
> [4/4] Calculating file sizes...
> => Found "@polkadot/wasm-crypto@6.3.1"
> info Reasons this module exists
>    - "@phala#sdk" depends on it
>    - Hoisted from "@phala#sdk#@polkadot#wasm-crypto"
> => Found "@polkadot/util-crypto#@polkadot/wasm-crypto@6.4.1"
> info This module exists because "@phala#sdk#@polkadot#util-crypto" depends on it.
> ```
> 
> This leads to errors while using api
> 
> ```
> @polkadot/wasm-crypto has multiple versions, ensure that there is only one installed.
> Either remove and explicitly install matching versions or dedupe using your package manager.
> The following conflicting packages were found:
>         cjs 6.3.1       node_modules/@polkadot/wasm-crypto/cjs
>         cjs 6.4.1       node_modules/@polkadot/util-crypto/node_modules/@polkadot/wasm-crypto/cjs
> @polkadot/wasm-bridge has multiple versions, ensure that there is only one installed.
> Either remove and explicitly install matching versions or dedupe using your package manager.
> The following conflicting packages were found:
>         cjs 6.3.1       node_modules/@polkadot/wasm-bridge/cjs
>         cjs 6.4.1       node_modules/@polkadot/util-crypto/node_modules/@polkadot/wasm-bridge/cjs
> @polkadot/wasm-crypto-wasm has multiple versions, ensure that there is only one installed.
> Either remove and explicitly install matching versions or dedupe using your package manager.
> The following conflicting packages were found:
>         cjs 6.3.1       node_modules/@polkadot/wasm-crypto-wasm/cjs
>         cjs 6.4.1       node_modules/@polkadot/util-crypto/node_modules/@polkadot/wasm-crypto-wasm/cjs
> @polkadot/wasm-util has multiple versions, ensure that there is only one installed.
> Either remove and explicitly install matching versions or dedupe using your package manager.
> The following conflicting packages were found:
>         cjs 6.3.1       node_modules/@polkadot/wasm-util/cjs
>         cjs 6.4.1       node_modules/@polkadot/util-crypto/node_modules/@polkadot/wasm-util/cjs
> ```

I fix it for the repo's example. Do you mean it?

**Comment by l00k** [comment]
I mean published `@phala/sdk` package.
I'm using it in `devphase` as direct dependency. 
Also I'm not specifying any other `@polkadot` package (expect `@polkadot/api-contract@^9.8.2` ) depenendency. So I suppose error comes from sdk package.

**Comment by Leechael** [comment]
It should temporarily fix in v0.3.8, so close this issue right now.

But we need to improve the dependencies declaration, we will look for a way to improve that. Let's move that into separated topic.

**Comment by Leechael** [comment]
Sorry, a mistake, so we released v0.3.9 for this fix.
